types/graphql-errors/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for graphql-errors 2.1
+// Project: https://github.com/kadirahq/graphql-errors
+// Definitions by: Matías Olivera <https://github.com/MatiasOlivera>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import { GraphQLSchema } from 'graphql';
+
+export type handlerFunction = (err: Error) => Error;
```

Comment:
style note: types usually start with an uppercase letter. Maybe there's a good reason to use lowercase 'h' here though?

Reply:
I'll fix it, thanks for letting me know.